feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/ConversationViewModel.kt

```diff
@@ -261,6 +270,9 @@ internal open class ConversationViewModel(
 
     val serverUrl = serverUrlStateFlow(serverConfigurationService)
 
+    private val _selectedFileUri = MutableStateFlow<Uri?>(null)
+    private val _selectedFileName = MutableStateFlow<String>("")
```

Comment:
Can you explain why we need the fileUri and name as mutable flows here? Wouldn't it be sufficient to pass them to the `uploadFileOrImage` as parameters?

Reply:
Adapted as you suggested